.gitignore

```diff
@@ -0,0 +1,72 @@
+node_modules/
+
+# Editors
+.vscode
+
+# Logs
+logs
```

Comment:
what makes this folder?

Reply:
I reckon it was auto-generated, I remove it for now.